docker/importer/volume/surveillance-mediator/case-report/endpoint-resource-resolver-dhis.json

```diff
@@ -82,7 +104,29 @@
-    "patcourse_dateonset": "{ 'dataElement': constants.dataElements.patcourse_dateonset, 'value': requestBody.data.observations[4].value.valueDate }"
+    "patcourse_dateonset": "{ 'dataElement': constants.dataElements.patcourse_dateonset, 'value': requestBody.data.observations[4].value.valueDateTime }",
+    "patcourse_admit": "{'dataElement': constants.dataElements.patcourse_admit, 'value': $lookup(constants.yesNoUnknownMap, requestBody.data.observations[9].value.valueCodeableConcept.coding[0].code) }",
+    "patcourse_vent": "{'dataElement': constants.dataElements.patcourse_vent, 'value': $lookup(constants.yesNoUnknownMap, requestBody.data.observations[11].value.valueCodeableConcept.coding[0].code) }",
+    "patcourse_presHCF": "{'dataElement': constants.dataElements.patcourse_presHCF,  'value': requestBody.data.encounters[1].period.start }",
+    "patcourse_iso": "{'dataElement': constants.dataElements.patcourse_iso, 'value': $lookup(constants.yesNoUnknownMap, requestBody.data.observations[13].value.valueCodeableConcept.coding[0].code) }",
+    "patcourse_dateiso": "{'dataElement': constants.dataElements.patcourse_dateiso, 'value': requestBody.data.observations[13].value.component[0].valueDateTime}",
+    "patinfo_occuhcw": "{'dataElement': constants.dataElements.patinfo_occuhcw 'value': $lookup(constants.yesNoUnknownMap, requestBody.data.observations[14].value.valueCodeableConcept.coding[0].code) }",
+    "expo_case_location": "{'dataElement': constants.dataElements.expo_case_location, 'value': request.data.observations[17].value.component[4].valueCodeableConcept.coding[0].code}",
+    "expo_visit_healthcare": "{'dataElement': constants.dataElements.expo_visit_healthcare, 'value': $lookup(constants.yesNoUnknownMap, requestBody.data.observations[16].value.valueCodeableConcept.coding[0].code) }",
+    "expo_contact_case": "{'dataElement': constants.dataElements.expo_contact_case, 'value': $lookup(constants.yesNoUnknownMap, requestBody.data.observations[17].value.valueCodeableConcept.coding[0].code) }",
+    "expo_ID1": "{'dataElement': constants.dataElements.expo_ID1, 'value': request.data.observations[17].value.component[1].valueString}",
+    "expo_travel_country1": "{'dataElement': constants.dataElements.expo_travel_country1, 'value': request.data.observations[15].value.valueCodeableConcept.coding[0].code }",
+    "patcourse_icu": "{'dataElement': constants.dataElements.patcourse_icu, 'value': $lookup(constants.yesNoUnknownMap, requestBody.data.observations[10].value.valueCodeableConcept.coding[0].code) }",
+    "patcourse_ecmo": "{'dataElement': constants.dataElements.patcourse_ecmo, 'value': $lookup(constants.yesNoUnknownMap, requestBody.data.observations[12].value.valueCodeableConcept.coding[0].code) }",
+    "comcond_preg": "{'dataElement': constants.dataElements.comcond_preg, 'value': 'PREGNANCY' in request.data.conditions ? 'PREGNANCY' : '' }",
```

Comment:
the logic here will not work. `request.data.conditions` is an array containing objects not strings

Reply:
@bradsawadye What is the structure of the objects?